src/sas/sasview/__init__.py

```diff
@@ -1,5 +1,5 @@
 from distutils.version import StrictVersion
-__version__ = "5.0.5-alpha.1"
+__version__ = "5.0.5a1"
```

Comment:
This fix is already being addressed in #1874 and should probably be removed.

Reply:
Of course. I just hijacked it to be able to run this branch and test it. Otherwise, it requires manual modification